src/config/select.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
I guess we can if we rather import React.propTypes rather, sine we are not using ReactDom for rendering and not react